packages/app/src/overview/components/backup/backup.component.tsx

```diff
@@ -57,12 +57,14 @@ interface Props {
   lastBackupDate?: Date
   onBackupCreate: () => void
   onBackupRestore?: () => void
+  isBackupActionDisabled: boolean
```

Comment:
🟢 
In the Mudita Center project we don’t use `is` prefix for non-functional values. I add this topic to discuss at the front-end meeting.

Reply:
Resolved in 42e6390e76b22a3456b1ce5b1efdbf9da9d4e2e8